scripts/generate_activity_script/src/script.ts

```diff
@@ -99,7 +139,7 @@ async function bondRandomAmount(
   );
   if (Number(IBCDenomBalance.amount) === 0) {
     return {
-      mode: MODE.BOND,
+      mode: NeutronAction.BOND,
```

Comment:
Please throw exception everywhere where you think your methods fails its execution

Reply:
Should we handle them in `main` or just let them to kill the process?